dmutils/email.py

```diff
@@ -231,3 +232,95 @@ def decode_invitation_token(encoded_token):
 
 def token_created_before_password_last_changed(token_timestamp, user_timestamp):
     return token_timestamp < user_timestamp
+
+
+class DMNotifyClient(object):
```

Comment:
Was there some discussion about the best place for this to live?  It seems a bit odd having this class in the `email.py` file alongside all the Mandrill stuff that isn't in its own `DMMandrill...` class, feels like this could be a separate thing altogether maybe.